estela-web/src/components/SpiderDetailPage/index.tsx

```diff
@@ -707,9 +707,7 @@ export class SpiderDetailPage extends Component<RouteComponentProps<RouteParams>
                                             <Col className="float-right">
                                                 <Button
                                                     onClick={() => {
-                                                        history.push(
-                                                            `/projects/${this.projectId}/spiders/${this.spiderId}/jobs/create`,
-                                                        );
+                                                        history.push(`/projects/${this.projectId}/jobs`);
```

Comment:
Instead to redirect to this route, I think we can show the window to create a job

Reply:
Solved